lib/block-patterns.php

```diff
@@ -62,3 +62,41 @@
 						<!-- /wp:columns -->',
 	)
 );
+
+
+// Buttons scoped pattern to test with Placeholder.
+register_block_pattern(
```

Comment:
@kjellr can we start thinking for which blocks we might want to have bundled patterns for? It doesn't have to be many, but it'd be good to be able to not rely on dummy patterns for testing and refining the UI.

Reply:
Also to start thinking about: Looking forward to implementing this with template parts, Im guessing we should probably register ~2 header and 2 footer patterns in Gutenberg itself when the time comes?  So that the patterns selection/transform can be used even if the theme or other plugins aren't registering any.  Perhaps they should be new/unique?  Or maybe we take from existing designs like tt1-blocks template parts? 🤔

Reply:
> @kjellr can we start thinking for which blocks we might want to have bundled patterns for?

Sounds good. Once the new patterns from https://github.com/WordPress/gutenberg/issues/28921 are in, we can evaluate which other blocks could use some patterns too. 👍